### PR TITLE
No multiple parents

### PR DESCRIPTION
**Pull request by emmanuelmathot**
**Related Issue(s):** #1230

**Proposed Changes:**

1. Clarify unique parent link requirement
2. Added collection relation type for link in catalog and collection
3. Moved the STAC structural relations in common metadata spec

**PR Checklist:**

- [X] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [X] This PR has **no** breaking changes.
- [X] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md)
      **or** a CHANGELOG entry is not required.
- [ ] This PR affects the [STAC API spec](https://github.com/radiantearth/stac-api-spec),
      and I have opened issue/PR #XXX to track the change.

### CODE REVIEW

**Comment by m-mohr** [comment]
I don't think this solves #1236 and #1273, because the issues mostly look at the collection relation type while here we are looking at the parent relation type.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The `keywords` field known from Collections is available in common metadata. ([#1187](https://github.com/radiantearth/stac-spec/issues/1187))
 - The `license` field additionally supports SPDX expressions and the value `other`.
 - The `roles` field known from Assets and Providers is available in common metadata. ([#1267](https://github.com/radiantearth/stac-spec/issues/1267))
+- `collection` relationship type for link in catalog and collection
+- Clarification on unique parent link requirement
```
Comment (m-mohr) [inline CHANGELOG.md:16]: Should be moved to Changed or Fixed, I think.

File: best-practices.md [hunk 2]
```diff
@@ -657,6 +657,13 @@ So if you are writing a STAC client it is recommended to start with just support
 turn, if your data is published online publicly or for use on an intranet then following these recommendations will ensure
 that a wider range of clients will work with it.
 
+#### Unique Parent Link
+
+STAC catalogs do not support multiple parents. Links with `rel` of type `parent` and `collection` must be unique.
```
Comment (m-mohr) [inline best-practices.md:662]: > STAC catalogs do not support multiple parents.

This sounds like a requirement but is in best practices. Should this be moved to the spec?

Comment (m-mohr) [reply best-practices.md:662]: Also, there can be multiple parent links for different encodings (e.g. application/json and text/html). We should clarify that this only applies for STAC-specific media types.

**Review by m-mohr — CHANGES_REQUESTED** [review]
See comments

**Comment by emmanuelmathot** [comment]
This PR is set at the agenda of the next STAC meetup along with the collections discussions.

**Comment by emmanuelmathot** [comment]
Discussed in STAC meetup: 

- Single STAC parent in spec
- Leave collection out of PR

File: best-practices.md (hunk 2 above)
Comment (emmanuelmathot) [reply best-practices.md:662]: moved to spec

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Several typos and minor language changes
 - Clarified that collection IDs should be unique across all collections in the corresponding root catalog.
+- Clarification on unique parent link requirement ([#1279]([https:](https://github.com/radiantearth/stac-spec/pull/1279)))
```
Comment (m-mohr) [inline CHANGELOG.md:38]:
```suggestion
- Clarification on unique parent link requirement ([#1279](https://github.com/radiantearth/stac-spec/pull/1279))
```

File: catalog-spec/catalog-spec.md [hunk 4]
```diff
@@ -85,13 +85,14 @@ For a full discussion of the situations where relative and absolute links are re
...
+| self       | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Catalog file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.                  |
+| root       | STRONGLY RECOMMENDED. URL to the root STAC Catalog or [Collection](../collection-spec/README.md). Catalogs should include a link to their root, even if it's the root and points to itself.                                                                                      |
+| parent     | URL to the parent STAC entity (Catalog or Collection). Non-root Catalogs should include a link to their parent.                                                                                                                                                                  |
+| collection | URL to a parent Collection. *Absolute* URLs should be used whenever possible. The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Catalog. A link with this `rel` type is *required* if the `collection` field in properties is present. |
```
Comment (m-mohr) [inline catalog-spec/catalog-spec.md:93]: I think the changelog should mention that we add the collection relation type for catalogs.

File: item-spec/item-spec.md [hunk 5]
```diff
@@ -193,11 +193,16 @@ This happens where there is not a clear official option, or where STAC uses an o
...
-| parent       | URL to the parent STAC entity (Catalog or Collection).                                                                                                                                                                                                                                       |
+| root         | URL to the root STAC entity (Catalog or Collection). ONLY ONE link with `root` relationship to STAC entity is allowed.                                                                                                                                                                       |
+| parent       | URL to the parent STAC entity (Catalog or Collection). ONLY ONE link with `parent` relationship to STAC entity is allowed.                                                                                                                                                                   |
```
Comment (m-mohr) [inline item-spec/item-spec.md:197]: I think we need to say here that a link to a JSON-version of a single STAC-entity is allowed. Multiple root or parent links can occur if they point to different encodings or language versions, but they should all point to the same entitiy (e.g. identified by the ID)

File: catalog-spec/catalog-spec.md [hunk 6]
```diff
@@ -85,13 +85,14 @@ For a full discussion of the situations where relative and absolute links are re
...
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| self       | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Catalog file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.                  |
+| root       | STRONGLY RECOMMENDED. URL to the root STAC Catalog or [Collection](../collection-spec/README.md). Catalogs should include a link to their root, even if it's the root and points to itself.                                                                                      |
+| parent     | URL to the parent STAC entity (Catalog or Collection). Non-root Catalogs should include a link to their parent.                                                                                                                                                                  |
```
Comment (m-mohr) [inline catalog-spec/catalog-spec.md:92]: The link restrictions that we add to items must also be present here, I think.

File: collection-spec/collection-spec.md [hunk 7]
```diff
@@ -270,15 +270,16 @@ It is recommended to use the official
...
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| self         | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Collection file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.                  |
+| root         | URL to the root STAC entity (Catalog or Collection). Collections should include a link to their root, even if it's the root and points to itself.                                                                                                                                   |
+| parent       | URL to the parent STAC entity (Catalog or Collection). Non-root Collections should include a link to their parent.                                                                                                                                                                  |
```
Comment (m-mohr) [inline collection-spec/collection-spec.md:277]: The link restrictions that we add to items must also be present here, I think.

File: collection-spec/collection-spec.md [hunk 8]
```diff
@@ -270,15 +270,16 @@ It is recommended to use the official
...
+| collection   | URL to a parent Collection. *Absolute* URLs should be used whenever possible. The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Collection. A link with this `rel` type is *required* if the `collection` field in properties is present. |
```
Comment (m-mohr) [inline collection-spec/collection-spec.md:278]: I think the changelog should mention that we add the collection relation type for collections.
It can point to issue https://github.com/radiantearth/stac-spec/issues/1236

**Comment by emmanuelmathot** [comment]
@m-mohr There was a table merge with previous PR #1259. Please make a new review.

File: catalog-spec/catalog-spec.md [hunk 9]
```diff
@@ -85,13 +85,14 @@ For a full discussion of the situations where relative and absolute links are re
...
+| root       | STRONGLY RECOMMENDED. URL to the root STAC entity (Catalog or [Collection](../collection-spec/README.md)). Catalogs should include a link to their root, even if it's the root and points to itself. ONLY ONE link with `root` relationship and `application/json` type is allowed. Other version of the root link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID). | application/json                                     |
+| parent     | URL to the parent STAC entity (Catalog or [Collection](../collection-spec/README.md)). Non-root Catalogs should include a link to their parent. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other version of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).                                                  | application/json                                     |
```
Comment (gadomski) [inline catalog-spec/catalog-spec.md:92]:
```suggestion
| parent     | URL to the parent STAC entity (Catalog or [Collection](../collection-spec/README.md)). Non-root Catalogs should include a link to their parent. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other versions of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).                                                  | application/json                                     |
```

File: collection-spec/collection-spec.md [hunk 10]
```diff
@@ -270,15 +270,16 @@ It is recommended to use the official
...
+| root         | STRONGLY RECOMMENDED. URL to the root STAC entity ([Catalog](../catalog-spec/README.md) or Collection). Collections should include a link to their root, even if it's the root and points to itself. ONLY ONE link with `root` relationship and `application/json` type is allowed. Other version of the root link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID). | application/json                                     |
+| parent       | URL to the parent STAC entity ([Catalog](../catalog-spec/README.md) or Collection). Non-root Collections should include a link to their parent. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other version of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).                                                  | application/json                                     |
```
Comment (gadomski) [inline collection-spec/collection-spec.md:277]:
```suggestion
| parent       | URL to the parent STAC entity ([Catalog](../catalog-spec/README.md) or Collection). Non-root Collections should include a link to their parent. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other versions of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).                                                  | application/json                                     |
```

File: item-spec/item-spec.md [hunk 11]
```diff
@@ -210,13 +210,18 @@ It is recommended to use the official
...
+| root         | URL to the root STAC entity (Catalog or Collection). `type` field must be `application/json`. ONLY ONE link with `root` relationship and `application/json` type is allowed. Other version of the root link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).       | application/json                                     |
+| parent       | URL to the parent STAC entity (Catalog or Collection). `type` field must be `application/json`. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other version of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID). | application/json                                     |
```
Comment (gadomski) [inline item-spec/item-spec.md:217]:
```suggestion
| parent       | URL to the parent STAC entity (Catalog or Collection). `type` field must be `application/json`. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other versions of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID). | application/json                                     |
```

**Review by gadomski — CHANGES_REQUESTED** [review]

File: catalog-spec/catalog-spec.md [hunk 12]
```diff
@@ -85,14 +85,14 @@ For a full discussion of the situations where relative and absolute links are re
...
+| root       | STRONGLY RECOMMENDED. URL to the root STAC entity (Catalog or [Collection](../collection-spec/README.md)). `type` field must be `application/json`. Catalogs should include a link to their root, even if it's the root and points to itself. ONLY ONE link with `root` relationship and `application/json` type is allowed. Other version of the root link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID). |
+| parent     | URL to the parent STAC entity (Catalog or [Collection](../collection-spec/README.md)). `type` field must be `application/json`. Non-root Catalogs should include a link to their parent. ONLY ONE link with `parent` relationship and `application/json` type is allowed. Other version of the parent link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).                                                  |
```
[the remark this answers is not in the document]
Comment (emmanuelmathot) [reply catalog-spec/catalog-spec.md]: one common file for all link object right? in `item-spec` folder?

[the remark this answers is not in the document]
Comment (emmanuelmathot) [reply catalog-spec/catalog-spec.md]: I would put it in common-metadata.md. wdyt?

[the remark this answers is not in the document]
Comment (emmanuelmathot) [reply catalog-spec/catalog-spec.md]: There are many relationship types that are specific to each STAC entity type that there would be anyway a specific table in addition of a "common" table in the common metadata. @m-mohr Do you think it is worth it?

[the remark this answers is not in the document]
Comment (m-mohr) [reply catalog-spec/catalog-spec.md]: Maybe not? Let's do it separately, e.g. in #1203.
Having that said, I think the content in the table cells should still be shortened and the long version be put below the table similar to how it's done for other fields etc.

[the remark this answers is not in the document]
Comment (emmanuelmathot) [reply catalog-spec/catalog-spec.md]: I gave it a try in 9d862e0db241b8ad249b96e55f7a3240a0d97336, let me know if that suits your change request otherwise, I roll it back

File: item-spec/common-metadata.md [hunk 13]
```diff
@@ -212,3 +213,61 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+#### Root and parent relation
+
+The `root` and `parent` relations are used to link to the root and parent STAC entity, which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
+**ONLY ONE** link with `root` or `parent` relationship and `application/json` type is allowed.
+Other version of the `root` or `parent` link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).
+
+#### Collection relation
+
+The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Collection.
+A link with this `rel` type is *required* for STAC item if the `collection` field in properties is present.
```
Comment (m-mohr) [inline item-spec/common-metadata.md:252]: This line should probably be in the Item Spec where the collection field is defined.

File: item-spec/common-metadata.md [hunk 14]
```diff
@@ -212,3 +213,61 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+#### Self relation
+
+The `self` relation is used to link to the STAC entity itself.
+This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.
+
+#### Root and parent relation
+
+The `root` and `parent` relations are used to link to the root and parent STAC entity, which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
+**ONLY ONE** link with `root` or `parent` relationship and `application/json` type is allowed.
+Other version of the `root` or `parent` link can be added with different `type` field as long as they point the same conceptual entity (e.g. identified by the ID).
```
Comment (m-mohr) [inline item-spec/common-metadata.md:247]:
```suggestion
The `root` and `parent` relations are used to link to the root and parent STAC entity,
which is either a [Catalog](../catalog-spec/README.md) or a [Collection](../collection-spec/README.md).
Conceptually, STAC entities SHALL have no more than one parent entity.
As such, STAC entities also can have no more than one root entity.
Therefore, there's usually just one link with `root` or `parent` relationship
unless different variations of the same conceptual entity exist (identified by the ID).
Different variations could be:
- a different encoding (see the `type` property), e.g. a HTML version in addition to JSON
- a different language (see the `hreflang` property). e.g. a German version in addition to English
```

Trying to clarify a bit more and add that different languages could also be a variations.

File: item-spec/common-metadata.md [hunk 15]
```diff
@@ -212,3 +213,61 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+
+#### Collection relation
+
+The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Collection.
+A link with this `rel` type is *required* for STAC item if the `collection` field in properties is present.
+
+#### Item relation
+
+All Items linked from a Collection MUST refer back to its Collection with the [`collection` relation type](#collection-relation).
```
Comment (m-mohr) [inline item-spec/common-metadata.md:256]: This seems a bit off, i.e. there's context missing. The header is Item reltation, but it discussed the collection relation.

File: item-spec/common-metadata.md [hunk 16]
```diff
@@ -212,3 +213,61 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+
+### Additional relations
+
+| Type         | Description                                                                                                      | Media Type       |
+| ------------ | ---------------------------------------------------------------------------------------------------------------- | ---------------- |
+| license      | The license URL(s) for the Item SHOULD be specified if the `license` field is **not** a SPDX license identifier. | Any              |
+| derived_from | URL to a STAC Entity that was used as input data in the creation of this Entity.                                 | application/json |
```
Comment (m-mohr) [inline item-spec/common-metadata.md:263]: Maybe we should move the derived_from to the best practices and license back to the common metadata field. But that doesn't need to be part of this PR necessarily.

File: catalog-spec/catalog-spec.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (m-mohr) [reply catalog-spec/catalog-spec.md]: Thanks. It's a good start. Maybe we should at some point split the common metadata, but we can do that later.

File: item-spec/json-schema/item.json [hunk 17]
```diff
@@ -204,21 +204,6 @@
               "title": "Link title",
               "type": "string"
             }
-          },
```
Comment (m-mohr) [inline item-spec/json-schema/item.json:207]: The removal is likely unintentional?

File: item-spec/item-spec.md [hunk 18]
```diff
@@ -299,15 +282,6 @@ As detailed above, Items contain properties, which are the main source of metada
 extensions can add further property fields as well. Any property that can be specified for an Item can also be specified for
 a specific asset. This can be used to override a property defined in the Item, or to specify fields for which there is no
 single value for all assets.
-If a property is defined in the Item Properties, it is the default value for all assets in the Item.
```
Comment (m-mohr) [inline item-spec/item-spec.md:302]: Is this removal intentional?

File: best-practices.md [hunk 19]
```diff
@@ -673,19 +673,27 @@ with the `type` field) to communicate the structure and content of related entit
...
 Being liberal with the `links` also means that it's ok to have repeated links with the same `href`. For example the
 `parent` and `root` relation types will point at the same file when the child is directly below the root, and it is
 recommended to include both. 
 
+#### Derived from relation (`derived_from`)
+
+A full provenance model is far beyond the scope of STAC,
+and the goal is to align with any good independent spec that comes along for that.
+But the derived_from field is seen as a way to encourage fuller specs and at least start a linking
+structure that can be used as a jumping off point for more experiments in provenance tracking*
```
Comment (gadomski) [inline best-practices.md:695]:
```suggestion
But the `derived_from` field is seen as a way to encourage fuller specs and at least start a linking
structure that can be used as a jumping off point for more experiments in provenance tracking.
```

File: collection-spec/collection-spec.md [hunk 20]
```diff
@@ -263,30 +263,14 @@ For a full discussion of the situations where relative and absolute links are re
...
-it is **REQUIRED** that Items linked from a Collection MUST refer back to its Collection
-with the [`collection` relation type](../item-spec/item-spec.md#relation-types).
+All the [common relation types](../item-spec/common-metadata.md#relation-types) can be used in Collection.
+A `self` and a `root` links are STRONGLY RECOMMENDED.
+Non-root Collections SHOULD include a `parent` link to their parent.
+
+> \[!NOTE] The STAC Catalog specification requires a link to at least one `item` or `child` Catalog.
+> This is *not* a requirement for Collections, but *recommended*. In contrast to Catalogs,
+> it is **REQUIRED** that Items linked from a Collection MUST refer back to its Collection
+> with the [`collection` relation type](../item-spec/item-spec.md#relation-types).
```
Comment (gadomski) [inline collection-spec/collection-spec.md:273]:
```suggestion
> with a link with the [`collection` relation type](../item-spec/item-spec.md#relation-types).
```

File: item-spec/json-schema/item.json (hunk 17 above)
Comment (emmanuelmathot) [reply item-spec/json-schema/item.json]: I think this is an error when merging with dev branch

File: item-spec/item-spec.md (hunk 18 above)
Comment (emmanuelmathot) [reply item-spec/item-spec.md]: again an error with the dev merge

File: item-spec/common-metadata.md [hunk 21]
```diff
@@ -212,3 +217,53 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
 and for WorldView-3 the Multispectral 20° off-nadir value of 1.38.
+
+## Relation types
+
+STAC Entities use a variety of `rel` types in the link object,
+to describe the exact nature of the link between the STAC object and the entity it is linking to.
+It is recommended to use the official
+[IANA Link Relation Types](https://www.iana.org/assignments/link-relations/link-relations.xhtml) where possible.
+
+### STAC Structure relation
```
Comment (m-mohr) [inline item-spec/common-metadata.md:228]:
```suggestion
### Hierarchical relations
```

File: item-spec/common-metadata.md [hunk 22]
```diff
@@ -212,3 +217,53 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+| self       | *Absolute* URL to the location that the STAC file can be found online, if available.                                | application/json                                     |
+| root       | URL to the root STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).   | application/json                                     |
+| parent     | URL to the parent STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)). | application/json                                     |
+| child      | URL to a child STAC entity ([Catalog](../catalog-spec/README.md) or [Collection](../collection-spec/README.md)).    | application/json                                     |
+| collection | URL to the parent Collection. *Absolute* URLs should be used whenever possible.                                     | application/json                                     |
+| item       | URL to a STAC Item.                                                                                                 | application/geo+json (preferred) or application/json |
+
+#### Self relation
```
Comment (m-mohr) [inline item-spec/common-metadata.md:242]:
```suggestion
#### `self` relation
```

File: item-spec/common-metadata.md [hunk 23]
```diff
@@ -212,3 +217,53 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+
+The `self` relation is used to link to the STAC entity itself.
+This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.
+
+#### Root and parent relation
```
Comment (m-mohr) [inline item-spec/common-metadata.md:247]:
```suggestion
#### `root` and `parent` relation
```

File: item-spec/common-metadata.md [hunk 24]
```diff
@@ -212,3 +217,53 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+Conceptually, STAC entities SHALL have no more than one parent entity.
+As such, STAC entities also can have no more than one root entity.
+Therefore, there's usually just one link with `root` or `parent` relationship
+unless different variations of the same conceptual entity exist (identified by the ID).
+Different variations could be:
+
+- a different encoding (see the `type` property), e.g. a HTML version in addition to JSON
+- a different language (see the `hreflang` property). e.g. a German version in addition to English
+
+#### Child relation
```
Comment (m-mohr) [inline item-spec/common-metadata.md:260]:
```suggestion
#### `child` relation
```

File: item-spec/common-metadata.md [hunk 25]
```diff
@@ -212,3 +217,53 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
...
+
+The `child` relation is **ONLY** used to link a catalog or collection to a child catalog or collection.
+
+#### Collection and item relation
```
Comment (m-mohr) [inline item-spec/common-metadata.md:264]:
```suggestion
#### `collection` and `item` relation
```

File: item-spec/item-spec.md [hunk 26]
```diff
@@ -203,30 +202,14 @@ For a full discussion of the situations where relative and absolute links are re
...
-A more complete list of potential `rel` types and their meaning in STAC can be found in the [Using Relation 
-Types](../best-practices.md#using-relation-types) best practice. 
-
-##### derived_from
-
-*Note regarding the type `derived_from`: A full provenance model is far beyond the scope of STAC,
-and the goal is to align with any good independent spec that comes along for that.
-But the derived_from field is seen as a way to encourage fuller specs and at least start a linking
-structure that can be used as a jumping off point for more experiments in provenance tracking*
+All the [common relation types](../item-spec/common-metadata.md#relation-types) can be used in Item.
```
Comment (m-mohr) [inline item-spec/item-spec.md:205]: I think that's not correct. You can't use `item` in items.

Comment (m-mohr) [inline item-spec/item-spec.md:205]: I think that's not correct. You can't use `item` in items.

```suggestion
All [common relation types](../item-spec/common-metadata.md#relation-types) except for `item` can be used in Items.
```

**Review by m-mohr — APPROVED** [review]

**Review by gadomski — APPROVED** [review]
